extensions/hibernate-orm/runtime/pom.xml

```diff
@@ -65,6 +65,10 @@
             <groupId>org.hibernate.orm</groupId>
             <artifactId>hibernate-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.tool</groupId>
+            <artifactId>hibernate-tools-language</artifactId>
+        </dependency>
```

Comment:
Noooooooooooooooooooooooo :).

We definitely don't want this dependency at runtime.

Maybe it's time to split this into a runtime-dev module? /cc @phillip-kruger

Reply:
As far as I remember there was an issue with the hibernate extension with this. 
@alexey might know the details. Another option could be to use BuildTimeActions ? Is there a reason why this can not happen in the deployment class path ?

Reply:
Would marking the dependency as `optional` be a valid temporary workaround, while we figure out how to split the dev components to their own module? We already depend on e.g. `quarkus-devui` optionally in the runtime module, so that seems like it could be fine for now.

Reply:
@aloubyansky any news on the limitation preventing us from separating Hibernate's dev mode in a dedicated module? I'd like to get this integrated, if you think it will take a while we could try Guillaume's suggestion of creating an empty runtime-dev module instead.

Reply:
So we can try to split the Hibernate runtime-dev module now? I'm happy to give that a try, or should we wait for this feedback before going for more dev only artifacts?

Reply:
Ok, there were more https://github.com/quarkusio/quarkus/compare/main...aloubyansky:quarkus:dev-only-deps-hibernate

Reply:
I couldn't merge those because they were breaking Gradle IT. We may just try rebasing them now.

Reply:
Thanks, I've created https://github.com/quarkusio/quarkus/pull/49399 on top of those commits